src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,44 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
+
+  this.signal('loadBindingList');
+
+  this.container.find('#genes .glyphicon-remove')
+    .css('display', this.data.tracks.length == 1 ? 'none' : '');
+};
+
+/**
+ * Updates the track list for panel.
+ */
+genotet.BindingPanel.prototype.updateTrackList = function() {
```

Comment:
Sorry for my aggressive words. I am just taking your suggestions too seriously and assume that you have considered carefully. I tested everything carefully to make sure both the new feature and the whole system works well before I push every commit. I know that my design and structure are not perfect, but I did everything for functionality with reasons. Anyway, sorry again for the aggressive words and thank you very much for your time to review my code carefully.

By the way, maybe we could spend more time discussing the design before writing codes. I could write a design doc with the design of both objects and functions in detail. So that we could save much time discussing and unnecessary revision while reviewing codes. Think more before writing is what you always tell me.